frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -804,6 +804,24 @@ export const dashboardLogic = kea<dashboardLogicType>({
                 },
             ],
         ],
+        sortTilesByLayout: [
+            (s) => [s.layoutForItem],
+            (layoutForItem) => (tiles: Array<DashboardTile>) => {
+                return [...tiles].sort((a: DashboardTile, b: DashboardTile) => {
```

Comment:
Is it enough to only sort by `y`? Less for the future traveller to think about?

https://github.com/PostHog/posthog/blob/cb5bb94005f64cb5f45dad37ad3c23fecf0a6875/posthog/models/dashboard_tile.py#L134

Reply:
Hmm. We're primarily sorting on x here so I'm wondering if that backend code is even wrong.

Will test as I update this!

Reply:
This makes me think of Advent of Code. I never get X and Y the right way around 🤣

Reply:
Sorting by y is correct! Flipping :)